src/Resolver.hack

```diff
@@ -69,14 +69,14 @@ final class Resolver {
         $schema = $this->schema;
 
         if ($operation_name is nonnull) {
-            assert(
+            invariant(
```

Comment:
`assert` must be post HHVM 4.56

Reply:
This is [GraphQL\assert](https://github.com/mwildehahn/hack-graphql/blob/main/src/UserFacingError.hack#L5) which raises a `UserFacingError`.

Though I remember that there used to be a built-in `assert` in HHVM so maybe this conflicts with it and we need to rename it :(

Reply:
Ah, gotcha - will fix.